LICENSE-binary

```diff
@@ -232,19 +232,19 @@ com.google:guice:4.0
 com.google:guice-servlet:4.0
 com.google.api.grpc:proto-google-common-protos:1.0.0
 com.google.code.gson:2.9.0
-com.google.errorprone:error_prone_annotations:2.2.0
```

Comment:
Do we need upgrade to laster verison, 2.2.0 is too old.
![image](https://github.com/user-attachments/assets/1a932a80-aad7-498d-8c71-8800c4d88bf9)